### DISCUSSION TITLE
Getting Attempted to create a duplicate task reference name error even when no duplicate reference names in the Dynamically forked tasks

### DISCUSSION THREAD

**Discussion by v891** [Netflix/conductor#3568] [Q&A]
I occasionally see the below error during the execution of a workflow with Dynamic sub-workflows or tasks.

`No dynamic tasks could be created for the Workflow: abc.12/b6bf5cf6-5b6d-42d7-a17d-fb633a7d487e, Dynamic Fork Task: null/xyz_0e529f8c296d Attempted to create a duplicate task reference name: xyz_0e529f8c296d`
I'm unable to find the root cause of this error, and I would appreciate any insights or help in resolving it.

Additional information:

I'm using **v3.13.3** of the Conductor repository.
My database type is **postgres**.

I have checked the task reference names, and I'm not using the same name twice.
I would appreciate any help or suggestions on how to fix this issue. Thank you in advance!

**Comment by v891** [comment]
If anyone else runs into the same issue and comes across this post.
👉My issue got resolved after enabling the Workflow locking.

**Reply by thalvadzhiev** [reply]
Hey @v891 ! Did you resolve your issue by including `workflow.decider.locking.enabled=true` in config.properties? I'm still getting this issue even after including this option.
